### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by neil-tan**
Merging the rom-tensor hot-fix to master.

### CODE REVIEW

**Comment by mbartling** [comment]
@neil-tan Looks good, but can you do me a huge favor and make sure the cgen requirements link is the correct stable latest? I am away from my computer right now.

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+-e git+git@github.com:uTensor/utensor_cgen.git@367735d3b5f4f4fac7b4a84aceb1c6593f964460#egg=utensor_cgen
```
Comment (mbartling) [inline requirements.txt]: Is this the stable latest?

Comment (neil-tan) [reply requirements.txt]: @mbartling yes, same as the dev-branch. look forward to your extensions.

Comment (neil-tan) [reply requirements.txt]: Actually, I'm not sure.
https://github.com/uTensor/utensor_cgen/releases
@mbartling @dboyliao 
Can one of you please help to update this?

**Pull request merged**